railties/lib/rails/generators/rails/authentication/authentication_generator.rb

```diff
@@ -42,9 +46,9 @@ def configure_authentication_routes
       def enable_bcrypt
         if File.read(File.expand_path("Gemfile", destination_root)).include?('gem "bcrypt"')
           uncomment_lines "Gemfile", /gem "bcrypt"/
-          Bundler.with_original_env { execute_command :bundle, "install --quiet" }
```

Comment:
Worth to note that `execute_command` is public API, and if an application generator was doing `execute_command(:bundle)`, then this will no longer work (as `bin/bundle` will no longer exist).

However, in the documentation of `execute_command` it is mentioned that only `rails` or `rake` can be used. 
https://github.com/rails/rails/blob/0f9de98e869e82b726c0da62d9e5e36c2b0e9d8e/railties/lib/rails/generators/actions.rb#L458-L460

Reply:
I don't think it was intentional that these private methods were leaked into the API doc.

These were originally protected methods, and protected methods were leaked into the API doc whether intended or not.

When we changed the visibility of these methods to private, `# :doc:` was added to keep them visible to the API doc. d1daf4c31301f5f5917b877fd63a817f5f4608ed